.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
For me, it is not immediately clear what a bug in a specification means. So maybe we should make that explicit and maybe find a more intuitive name?

I think I would see a bug in a specification as something unclear, inconsistent, or contradictory with other parts of the specification.

Though there are also possible bugs in the hosting of the OTM5 specification, so we can name those as well.

Reply:
I agree with the classifiers you list here. I'll add those to the readme and the bug template.